src/karts/skidding.cpp

```diff
@@ -443,6 +443,9 @@ void Skidding::update(float dt, bool is_on_ground,
                                              bonus_speed, bonus_speed,
                                              bonus_force, bonus_time,
                                              /*fade-out-time*/ 1.0f);
+                    // Play custom character sound (weee!)
+                    m_kart->playCustomSFX(SFXManager::CUSTOM_SKID);
+                    m_kart->m_controller->handleZipper(true);                   
```

Comment:
If the goal of this PR is only to play a sound, then "handleZipper" should definitely not be called, a that may have side effects (such as notifying the server that you used a zipper, in multiplayer mode, which is definitely not intended)